add_trackList/add_trackList.py

```diff
@@ -0,0 +1,80 @@
+if not path.exists(dst_dir):
+    print('Create a symlink')
+    os.symlink(src_dir, dst_dir)
+
+#backup trackList in apollo directory
+shutil.copyfile(path.join('/app/data/other_species', gggsss, assembly_name, 'jbrowse/data', 'trackList.json'), path.join('/app/data/other_species', gggsss, assembly_name, 'jbrowse/data', 'trackList.json.bak'))
+
+#add bam to trackList
+print('add bam to trackList ...')
+label = os.path.splitext(os.path.basename(args.input_bam))[0]
+metadata = '{"metadata":{"Analysis provider": "i5k Workspace@NAL",\n"Analysis method": "https://github.com/NAL-i5K/NAL_RNA_seq_annotation_pipeline/",\n"Data source":"' + ','.join(Submission) + '",\n"Publication status":"Analysis: NA; Source data: see individual SRA accessions",\n"Track legend":"Dark red alignments: Mapped portion of read aligned to forward strand<br>Light red alignments: Spliced portion of read aligned to forward strand<br>Dark blue alignments: Mapped portion of read aligned to reverse strand<br>Light blue alignments: Spliced portion of read aligned to reverse strand<br>Red marking - deletion in the read relative to the reference<br>Green marking - insertion in the read relative to the reference<br>Yellow marking  - mismatch (hover over the mismatch to see what the snp is)"}, "type": "WebApollo\/View\/Track\/DraggableAlignments" }'
+proc_bam = subprocess.Popen(['add-bam-track.pl', '-i', args.input_track, '-o', args.input_track, '--bam_url', path.join('analyses', folder_name, args.input_bam), '--label', label[0:-10] + datetime.datetime.now().strftime("%Y-%m-%d"), '--category', 'RNA-Seq/Mapped Reads', '--config', metadata])
+proc_bam.communicate()
+
+#add bigwig to trackList
+print('add bigwig to trackList ...')
+metadata = '{"metadata":{"Analysis provider": "i5k Workspace@NAL",\n"Analysis method": "https://github.com/NAL-i5K/NAL_RNA_seq_annotation_pipeline/",\n"Data source":"' + ','.join(Submission) + '",\n"Publication status":"Analysis: NA; Source data: see individual SRA accessions",\n"Track legend":"This track represents an X-Y plot of RNA-Seq coverage."} }'
+proc_bw = subprocess.Popen(['add-bw-track.pl', '-i', args.input_track, '-o', args.input_track, '--bw_url', path.join('analyses', folder_name, args.input_bigwig), '--label', label[0:-10] + datetime.datetime.now().strftime("%Y-%m-%d") + '_coverage', '--plot', '--category', 'RNA-Seq/Coverage Plots', '--pos_color', "#00BFFF", '--config', metadata])
+proc_bw.communicate()
+
+#add juntion reads to trackList
+print('add junction reads  to trackList ...')
+metadata = '{"style":{"showLabels": false}, "metadata": {"Analysis provider": "i5k Workspace@NAL",\n"Analysis method": "https://github.com/NAL-i5K/NAL_RNA_seq_annotation_pipeline/",\n"Data source":"' + ','.join(Submission) + '",\n"Publication status":"Analysis: NA; Source data: see individual SRA accessions",\n"Track legend":"Intronic junction reads generated by Hisat2 aligner and regtools"}, "category":"RNA-Seq/Intronic splice junctions" }'
```

Comment:
@HsiuKangHuang I changed my mind - let's change `Intronic junction reads generated by Hisat2 aligner and regtools` to `Junction reads generated by Hisat2 aligner and regtools` and `RNA-Seq/Intronic splice junctions` to `RNA-Seq/Splice junctions`